babel.config.js

```diff
@@ -101,14 +103,12 @@ module.exports = function(api) {
       ["@babel/plugin-proposal-nullish-coalescing-operator", { loose: true }],
 
       convertESM ? "@babel/transform-modules-commonjs" : null,
+      isStandalone ? ["@babel/transform-for-of", { assumeArray: true }] : null,
```

Comment:
I'd prefer that we also apply this transform when running tests outside of `@babel/standalone`, otherwise we risk introducing a `for-of` with a `Map` which isn't caught by the tests.

Reply:
Actually, this probably already breaks part of `@babel/core` and all the modules plugins.

Reply:
That's an interesting usage of this plugin! I will try it later.

Reply:
Yup, it doesn't work:

https://babeljs.io/repl/build/13296#?browsers=Chrome%2023&build=&builtIns=false&spec=false&loose=false&code_lz=JYWwDg9gTgLgBAbzgDzgXzgMyhEcBEARvgNwBQZApspLHAG4CGUcjcAvCiUA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=es2015&prettier=false&targets=&version=7.7.4%2Bpr.10823&externalPlugins=